code/go/0chain.net/smartcontract/storagesc/sc.go

```diff
@@ -180,7 +181,7 @@ func (ssc *StorageSmartContract) statDecr(name string) {
 
 // functions execution
 
-func (sc *StorageSmartContract) Execute(t *transaction.Transaction,
+func (ssc *StorageSmartContract) Execute(t *transaction.Transaction,
```

Comment:
Please avoid unrelated refactoring. Not only can it lead to hard to find errors and obscure the real changes being made, but it can also make merging the PR a nightmare.

I cannot see any real changes in sc.go, if so can you revert these changes.